# Ocuupancy grid specs
map_size: {x: 20, y: 20}
map_resolution: 0.1
map_publish_freq: 1
map_frame: "/odom"
map_reference_frame: "/base_footprint"
